addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -922,6 +922,23 @@
 			}
 		}
 		
+		"215"	//The Degreaser
+		{
+			"desp"			"The Degreaser: {positive}Requires 300 damage for airblast"
+	
+				"spawn"
+				{
+					"Tags_Airblast"
+					{
+						"override"		"pyro-primary-airblast"
+						"target"		"primary"
```

Comment:
Don't think this line is needed as `target` is already set, unchanged from primary pyro